ThaDeveloper/FastFoodFast_API
# #160237591 Unittest

#### What does this PR do?
Test the API endpoints
#### Description of Task to be completed?
Have the tests for the different app functionalities
#### How should this be manually tested?
- `$ git clone https://github.com/ThaDeveloper/FastFoodFast_API`
- `$ cd FastFoodFast_API`
- Ensure you have installed Pytest and run `pytest`
#### Any background context you want to provide?
Built in flask microframework
#### What are the relevant pivotal tracker stories?
[#160237591](https://www.pivotaltracker.com/story/show/160237591)
